### ISSUE TITLE
`The passed save_path is not a valid checkpoint`

### ISSUE THREAD

**Issue by ZZysh** [DeepLabCut/DeepLabCut#1793]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Bug description

I use the GUI (python -m deeplabcut) to create a maDLC project. and label the frames and then choose the dlcrnet_50 to creating training dataset. after that I move to the Train tab and select OK. however, the error exist

raise ValueError("The passed save_path is not a valid checkpoint

I think it might be the neuronet work which i sellect is false. I try the resnet_50 and it run normally.

### Operating System

win10

### DeepLabCut version

the latest version

### DeepLabCut mode

multi animal

### Device type

gpu

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/DeepLabCut/DeepLabCut/blob/master/CODE_OF_CONDUCT.md)

**Comment by MMathisLab** [comment]
be sure you pass the same shuffle ID to the train tab after you create the training set. It might be it did not creat (check the project folder). As otherwise, I cannot reproduce your result and DLCRNet runs for me

**Comment by MMathisLab** [comment]
@ZZysh see my comment in your next issue #1797 -- there is an issue with your project.

**Comment by ZZysh** [comment]
thanks a lot



---- 回复的原邮件 ----
| 发件人 | Mackenzie ***@***.***> |
| 日期 | 2022年04月24日 22:55 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.******@***.***> |
| 主题 | Re: [DeepLabCut/DeepLabCut] `The passed save_path is not a valid checkpoint` (Issue #1793) |

Closed #1793.
